cmd/fyne/internal/commands/package-darwin.go

```diff
@@ -20,25 +20,31 @@ type darwinData struct {
 	Category      string
 }
 
-func (p *Packager) packageDarwin() error {
+func (p *Packager) packageDarwin() (err error) {
 	appDir := util.EnsureSubDir(p.dir, p.name+".app")
 	exeName := filepath.Base(p.exe)
 
 	contentsDir := util.EnsureSubDir(appDir, "Contents")
 	info := filepath.Join(contentsDir, "Info.plist")
-	infoFile, _ := os.Create(info)
+	infoFile, err := os.Create(info)
+	if err != nil {
+		return errors.Wrap(err, "Failed to write plist template")
+	}
+	defer func() {
+		if r := infoFile.Close(); r != nil {
+			err = r
```

Comment:
This is also setting a shadow error instead of the return value...

Reply:
Nope. That's not how named returns and 'defer' works